### PR TITLE
Update oc run help

### PR DESCRIPTION
**Pull request by soltysh**
@openshift/cli-review ptal, this is a minor thing, but worth having I've been wanted to have that one in for a long, already.

### CODE REVIEW

File: pkg/cmd/cli/cmd/wrappers.go [hunk 1]
```diff
@@ -406,28 +406,41 @@ var (
...
+
+		# Start the nginx container using the default command, but use custom arguments (arg1 .. argN)
+		# for that command.
+		%[1]s run nginx --image=nginx -- <arg1> <arg2> ... <argN>
+
+		# Start the nginx container using a different command and custom arguments.
+		%[1]s run nginx --image=nginx --command -- <cmd> <arg1> ... <argN>
+
+		# Start the perl container to compute π to 2000 places and print it out.
+		%[1]s run pi --image=perl --restart=OnFailure -- perl -Mbignum=bpi -wle 'print bpi(2000)'
```
Comment (mfojtik) [inline pkg/cmd/cli/cmd/wrappers.go]: looks like a job for a job :-)

**Comment by mfojtik** [comment]
@soltysh while you are here, i always missed the example of using deployment generator over the deployment config (we override upstream), would be nice to have that here as well?

File: pkg/cmd/cli/cmd/wrappers.go (hunk 1 above)
Comment (soltysh) [inline pkg/cmd/cli/cmd/wrappers.go]: It's a job, but I'll update the description.

**Comment by soltysh** [comment]
I've also added a section about generators to our docs: https://github.com/openshift/openshift-docs/pull/4612

**Comment by mfojtik** [comment]
[test]

**Comment by soltysh** [comment]
@mfojtik makes sense, although I don't want to make that help too lengthy (it's already is quite packed). That's why I've updated the docs with list of supported generators.

**Comment by juanvallejo** [comment]
LGTM

**Comment by mfojtik** [comment]
[merge][severity:blocker]

**Comment by soltysh** [comment]
Flake https://github.com/openshift/origin/issues/12487
[merge][severity:blocker]

**Comment by soltysh** [comment]
Flake https://github.com/openshift/origin/issues/11452
[merge][severity:blocker]

**Pull request merged**